rand_distr/src/normal.rs

```diff
@@ -37,6 +37,15 @@ use crate::utils::ziggurat;
 #[derive(Clone, Copy, Debug)]
 pub struct StandardNormal;
 
+impl Distribution<f32> for StandardNormal {
+    #[inline]
+    fn sample<R: Rng + ?Sized>(&self, rng: &mut R) -> f32 {
+        // TODO: use optimal 32-bit implementation
+        let x: f64 = self.sample(rng);
+        x as f32
+    }
+}
```

Comment:
What is the motivation of this? To exercise the type change and optimize later? As of now, it does not seem worth to add this, when `dist.sample(&mut rng) as f32` is equivalent.

Reply:
Yes, an optimised impl can follow later. Even like this though it helps with some generic coding.